base/utils/R/write.config.utils.R

```diff
@@ -41,7 +41,7 @@ get.quantiles <- function(quantiles.tag) {
 ##' get sensitivity samples as a list
 ##'
 ##' @param pft Plant Functional Type
-##' @param env 
+##' @param env environment to retrieve the sample list from.
```

Comment:
I don't think this is an environment in the R sense, but rather a set of samples for parameters that are "environmental" (e.g. soil parameters) rather than specific to the plant functional type. 

Maybe the above `@param pft` should become "list of samples from plant functional types", then:

```suggestion
##' @param env list of samples from environment parameters
```

Reply:
I'll have a look at pft again.